### ISSUE TITLE
[Keyboard Navigation – Make Code - Import]controls like: 'Create new' present on 'pop up screen' are not accessible using keyboard.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3756]
User Experience:
If 'any control will not be accessible through keyboard then users who are dependent on keyboard for navigation will not be able to access that control and perform actions which are associated to that control.

Note: User credentials should NOT be included in the bug.

Repro Steps:

1. Launch application URL
2. Home page will open.
3. Navigate to "Import" button and select it.
4. Navigate through the controls appearing under "Import" dialog.
5. Navigate to "Your GitHub Repo" option and Select it.
6. Navigate and select "Sign In" control under Sign in with GitHub and navigate to all the controls of dialogue "Clone or create your own GitHub repo
7. Verify the issue.

Actual Result:
controls like: 'Create new' present on 'pop up screen' are not accessible using keyboard.

Expected Result:
All the interactive element should be accessible using keyboard as well as mouse.

Note:
Same issue exists on Firefox + NVDA & Chrome + JAWS.

WCAG Reference:
[MAS 2.1.1 - Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #HCL-MakeCode-New; #WCAG2.1.1; #Win10-Edge(Chromium);

[WCAG2.1.1.zip](https://github.com/microsoft/pxt-microbit/files/5595726/WCAG2.1.1.zip)
